spec/associations/active_record_extensions_spec.rb

```diff
@@ -166,6 +166,13 @@ class Book < ActiveRecord::Base
           expect(school.country).to eq(nil)
         end
 
+        it "doesn't interfere with AR's belongs_to arguments" do
+          allow(ActiveRecord::Base).to receive(:belongs_to).with(:country, nil)
+          allow(ActiveRecord::Base).to receive(:belongs_to).with(:country, nil, {})
```

Comment:
There are differences between ruby 2.x and 3.x when keyword argument is passed.

It perhaps is caused by this issue.

- https://github.com/rspec/rspec-mocks/issues/1492

Therefore, to avoid this, I use two `allow(ActiveRecord::Base).to receive(:belongs_to).with` method.

It's enough to verify that it doesn't interfere with AR's `belongs_to` arguments, so the third argument can be `{}` or missing.